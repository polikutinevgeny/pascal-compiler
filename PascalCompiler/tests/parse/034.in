var
   n: integer;

begin
   while n < 0 doo
end.