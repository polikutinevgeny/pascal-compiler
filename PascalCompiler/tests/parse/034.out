Expected Do, got Identifier at 5:16
